README.md

```diff
@@ -110,6 +110,10 @@ open connections are kept until the swarm is destroyed (using `close` or
 
 Unannounces the sync service & cleans up the underlying UDP socket. `cb` is called once this is complete.
 
+### sync.clear(cb)
```

Comment:
Did you tell me today that clear isn't used anymore?

Reply:
Yeah I had to reinstate it for #reasons. Although I dislike it

Reply:
okay im using gregor's approach instead which is to keep track of the time the component loaded and compare that with the old sync states